mailbox/api/src/main/java/org/apache/james/mailbox/model/PartContentDescriptorImpl.java

```diff
@@ -53,26 +55,15 @@ public MimePath path() {
     }
 
     public int hashCode() {
-        final int PRIME = 31;
-        int result = 1;
-        result = PRIME * result + ((path == null) ? 0 : path.hashCode());
-        return result;
+        return Objects.hash(path);
```

Comment:
not related to this 769e437ecb56e0fbe3f6e132f0d8088c6e4b8048 commit?

Reply:
That class need a fix around an if statement -> doing little refactoring at the same time.